src/serialization/sb2.js

```diff
@@ -504,10 +477,70 @@ const parseScratchObject = function (object, runtime, extensions, topLevel, zip)
             // the file name of the sound should be the soundID (provided from the project.json)
             // followed by the file ext
             const assetFileName = `${soundSource.soundID}.${ext}`;
-            soundPromises.push(deserializeSound(sound, runtime, zip, assetFileName)
-                .then(() => loadSound(sound, runtime, sprite))
```

Comment:
What's the reasoning for handling loadSound later (when parsing other target info) and not here?

Reply:
Sprites don't exist yet. `loadSound` needs a soundBank to store the loaded SoundPlayers on. Sprites have one, but I don't think we should create a Sprite yet at this point. Instead I have a follow up for this https://github.com/LLK/scratch-vm/pull/1972